src/EventPage/EventPage.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
```

Comment:
Let's move the stylesheet to a file called `src/Event/components/EventPage.style.js` and then import it into the React file. This will also involve creating two new folders called `screens` and `components`.